src/main/java/com/codicesoftware/plugins/hudson/PlasticSCM.java

```diff
@@ -356,6 +368,13 @@ public DescriptorImpl getDescriptor() {
         return (DescriptorImpl) super.getDescriptor();
     }
 
+    @Override
+    public boolean requiresWorkspaceForPolling() {
+        // If we poll on controller, we don't need a workspace
+        // If we poll on an agent, we do need a workspace
```

Comment:
These comments are redundant IMO, I'd prefer to remove them

Reply:
Noted, comment removed.